### PR TITLE
Update Bookmarks for TBA

### PR DESCRIPTION
**Pull request by SehrishHussain**
Added a note: There is no proper way of importing/exporting bookmarks in TBA.

@championquizzer Please review this PR

### CODE REVIEW

File: content/tbb/export-and-import-bookmarks/contents.lr [hunk 1]
```diff
@@ -42,3 +42,5 @@ The bookmarks HTML file you saved is now ready to be imported into another web b
 **Import data from another browser**
 > Bookmarks can be transferred from Firefox to Tor Browser. There are two ways to export and import bookmarks in Firefox: [HTML file](https://support.mozilla.org/en-US/kb/export-firefox-bookmarks-to-backup-or-transfer) or JSON file.
 After exporting the data from the browser, follow the above step to import the bookmark file into your Tor Browser.
+
+**Note:** Currently Tor Browser for Android has no proper way of importing/exporting bookmarks.
```
Comment (championquizzer) [inline content/tbb/export-and-import-bookmarks/contents.lr]: I would like to tweak the phrase a little bit and also acknowledge the bug:
**Note:** Currently, on [Tor Browser for Android](https://www.torproject.org/download/#android), there is no good way to export and import bookmarks. [Bug #31617](https://bugs.torproject.org/tpo/applications/tor-browser/31617)  
Raw markdown:  
`**Note:** Currently, on [Tor Browser for Android](https://www.torproject.org/download/#android), there is no good way to export and import bookmarks. [Bug #31617](https://bugs.torproject.org/tpo/applications/tor-browser/31617)`

Comment (SehrishHussain) [reply content/tbb/export-and-import-bookmarks/contents.lr]: Thank you. Yes it looks more elaborate and presentable now. I should make the changes in new branch and create a new PR. @championquizzer

**Comment by championquizzer** [comment]
@SehrishHussain, seems like you switched branches before making the changes and I think you will now have to squash your commits.

**Comment by SehrishHussain** [comment]
@championquizzer I have successfully squashed the two commits. Now I am going to force push my changes. Working on it.

**Comment by SehrishHussain** [comment]
Capture
I forced push from my branch **patch-2** to **master**, but I don't see changes anywhere here on Github. Can you help?

**Comment by gusgustavo** [comment]
Try to push patch-2, for example:
`git push --force origin patch-2`

**Comment by SehrishHussain** [comment]
Hi @gusgustavo I forced push and see PR here. Mind looking at this!
PS: I have spent almost two days figuring out git. It was frustrating and a bit fulfilling  at the same time.

**Comment by championquizzer** [comment]
LGTM. Thanks for working on this!   
Issue Ticket: https://gitlab.torproject.org/tpo/web/support/-/issues/180

**Review by gusgustavo — APPROVED** [review]

**Pull request merged**